config/database.yml

```diff
@@ -5,6 +5,7 @@
 #   gem 'postgresql'
 #
 default: &default
+  host: localhost
```

Comment:
should this line be under development only?

Reply:
You are right! There was an error on the build so I just added that by mistake! I just fixed it just now!